docs/src/modules/java/pages/ai-coding-assistant.adoc

```diff
@@ -114,6 +124,38 @@ it should answer with something like
 ...
 ----
 
+== Claude code
```

Comment:
Would be nice if they all used the same file and syntax (e.g. for importing other files), but that's currently not the case. https://pnote.eu/notes/agents-md/